test/attr.test.js

```diff
@@ -30,6 +31,22 @@ describe('#attr', () => {
       expect(wrapper.find('span')).to.not.have.attr('disabled')
     })
 
+    it('passes when attribute exists without a value', (wrapper) => {
+      expect(wrapper.find('article')).to.have.attr('itemscope')
+    })
+
+    it('passes when attribute exists with a falsey (but not false/null/undefined) value', (wrapper) => {
+      expect(wrapper.find('article')).to.have.attr('draggable')
+    })
+
+    it('passes when attribute exists but has value `false`', (wrapper) => {
+      expect(wrapper.find('article')).to.not.have.attr('hidden')
```

Comment:
wait - if the value is false, it absolutely "has" the attribute. jsx attributes can have either boolean value, otherwise how would you pass `PropTypes.bool` and pass both `true` and `false`?